R/subset_datarqst.R

```diff
@@ -15,9 +15,17 @@
 #' @param Species Character vector of six letter species codes. Include all species that are requested.
 #' @param filter Character vector of unique population species combinations (in the format PopID_Species).
 #' Include all unique population species combinations requested.
-#'
-#' @return Create 4 .csv files at the location of the standard data. Files will be given a unique number
-#' based on the date when the subset was performed.
+#' @param include_conflicting is FALSE. If include_conflicting = TRUE, individuals of conflicting species which
+#' were identified as a species contained in `Species` at least once are included in the subset of data. If
+#' include_conflicting = FALSE, these individuals are removed.
```

Comment:
A think we can simplify the help documentation here a bit.
```suggestion
#' @param If include_conflicting = TRUE, individuals with conflicting species information
#' are included in the subset of data. If
#' include_conflicting = FALSE (default), these individuals are removed.
```